ab0x.go

```diff
@@ -48,6 +48,13 @@ func init() {
 
 }
 
+func NewHandler() *webdav.Handler{
+	return &webdav.Handler{
+		FileSystem: FS,
+		LockSystem: webdav.NewMemLS(),
+	}
+}
+
```

Comment:
Did you see the comment at the top of the file?
```
// Code generated by fileb0x at "2022-06-10 22:55:21.84892167 +0300 EEST m=+0.038018428" from config file "b0x.yaml" DO NOT EDIT.
// modification hash(c06e09df5514d5bdf3c145144658221e.84893f7d7f6af7d7916db9fe20160151)
```

Reply:
Sorry, didn't saw it